client/src/components/styles/_forms.scss

```diff
@@ -11,13 +11,19 @@
   .forgot-password-form {
     // sass-lint:disable-block nesting-depth
     .btn-primary {
+
+      &:active:focus {
```

Comment:
Strange code. Should be comma here?

Reply:
nope, it's showed only when two states are active at the same time

Reply:
Than use scss format

Reply:
done